src/main/resources/META-INF/rewrite/apache-httpclient-5.yml

```diff
@@ -0,0 +1,453 @@
+#
+# Copyright 2023 the original author or authors.
+# <p>
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# <p>
+# https://www.apache.org/licenses/LICENSE-2.0
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+########################################################################################################################
+# Apache HttpClient 5.x
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.apache.httpclient5.UpgradeApacheHttpClient_5
+displayName: Migrate to ApacheHttpClient 5.x
+description: >
+  Migrate applications to the latest Apache HttpClient 5.x release. This recipe will modify an
+  application's build files, make changes to deprecated/preferred APIs, and migrate configuration settings that have
+  changes between versions.
+tags:
+  - apache
+  - httpclient
+recipeList:
+  - org.openrewrite.java.dependencies.ChangeDependency:
+      oldGroupId: org.apache.httpcomponents
+      oldArtifactId: httpclient
+      newGroupId: org.apache.httpcomponents.client5
+      newArtifactId: httpclient5
+      newVersion: 5.1.x
+      overrideManagedVersion: true
+  - org.openrewrite.java.apache.httpclient5.UpgradeApacheHttpClient_5_ClassMapping
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.apache.httpclient5.UpgradeApacheHttpClient_5_ClassMapping
+displayName: Migrate to ApacheHttpClient 5.x Classes Namespace from 4.x
+description: Mapping of all the compatible classes of ApacheHttpClient 5.x from 4.x.
+recipeList:
+  - org.openrewrite.java.dependencies.ChangeDependency:
+      oldGroupId: org.apache.httpcomponents
+      oldArtifactId: httpclient
+      newGroupId: org.apache.httpcomponents.client5
+      newArtifactId: httpclient5
+      newVersion: 5.1.x
+      overrideManagedVersion: true
+
+  - org.openrewrite.java.ChangePackage:
+      oldPackageName: org.apache.http.client.methods
+      newPackageName: org.apache.hc.client5.http.classic.methods
+      recursive: "False"
```

Comment:
[False seems to be the default](https://github.com/openrewrite/rewrite/blob/bb9f47a7947a6ad6ebbe9dcece329856f3d1963f/rewrite-java/src/main/java/org/openrewrite/java/ChangePackage.java#L56-L60C13), so I propose we remove this here and everywhere further down such that it stands out when actually true.
```suggestion
```

Reply:
I originally omitted the recursive: False, expecting to be the default value, but when I got weird results (because I was having ChangeType before ChangePackage), I got crazy and add it everywhere just to check if it was the reason... And then forgot about it 😅 
Gonna remove it now!